prac_07/myguitars.py

```diff
@@ -0,0 +1,46 @@
+from prac_07.guitar import Guitar
```

Comment:
if your working in the same file you can just write "from guitar import Guitar" if you wanted to make your code a little shorter :)